### PR TITLE
switch to the sphinx readthedocs theme 

### PR DESCRIPTION
**Pull request by Franko1307**
Addresses https://github.com/systemslab/popper/issues/311

### CODE REVIEW

**Comment by UjjwalAyyangar** [comment]
@Franko1307 Take a look at git commit `--amend`.  It's helpful when you want to add something to the same commit message (like "logo" in the above commits).

File: docs/conf.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from recommonmark.parser import CommonMarkParser
```
Comment (UjjwalAyyangar) [inline docs/conf.py]: I think this module needs to be vendored. (from https://github.com/rtfd/recommonmark).
Also, it needs to be added to the setup.py file.

**Comment by ivotron** [comment]
@UjjwalAyyangar no need to squash/rewrite commits on a PR branch, github does it automatically.

File: docs/conf.py (hunk 1 above)
Comment (ivotron) [reply docs/conf.py]: this PR is only touching the `docs/` folder, so it's separate from the `cli/` folder

**Comment by ivotron** [comment]
@Franko1307 awesome, looks great!

**Pull request merged**